### PR TITLE
4 files to handle the task 0 and 1 requirement(success in 3 tests)

### PR DESCRIPTION
**Pull request by LWSSIM**

### CODE REVIEW

File: formatter.c [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+	{
+		_putchar(*s);
+		s++;
+	}
+}
+/**
+ * print_percent - print '%'
+ * @args: passed '%'
+ */
+void print_percent(va_list args)
```
Comment (laribox) [inline formatter.c:79]: why did you add the va_list argument here why not just leave it empty

**Review by laribox — COMMENTED** [review]
the code seems perfect my idea what different from your as i intended to resolve the entire string first and then print it but your idea is also good

File: formatter.c (hunk 1 above)
Comment (LWSSIM) [reply formatter.c:79]: I tired leaving it empty. But it created some bugs.
Then i thought it can be left that way for consistency wiith the other functions.
